common/path_test.cc

```diff
@@ -29,6 +29,12 @@
 #include "common/util.h"
 #include "gtest/gtest.h"
 
+#if PLATFORM_WINDOWS
+#define WIN32_LEAN_AND_MEAN
```

Comment:
Can you wrap that in `#ifundef WIN32_LEAN_AND_MEAN` / `#endif`? Not sure if it applies here, but clangd complains if macros are redefined.

Reply:
Done. That should be pretty safe as we don't have any headers that define this.